README.md

```diff
@@ -600,6 +613,48 @@ config
   .init((Plugin, args) => new Plugin(...args));
 ```
 
+#### Config plugins: ordering before
+
+Specify that the current `plugin` context should operate before another named `plugin`.
+You cannot use `before()` and `after()` on the same plugin.
+
+```js
+config
+  .plugin(name)
+    .before(otherName)
+    
+// Example
+
+config
+  .plugin('html-template')
+    .use(HtmlWebpackTemplate)
```

Comment:
The indentation here is inconsistent with some of the other existing entries - this indents the `.use()` one level deeper than `.plugin()`, the others don't. Thoughts on which is preferred?